api/src/main/java/org/hawkular/apm/api/services/ServiceDeploymentService.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2015-2017 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.apm.api.services;
+
+import java.util.List;
+
+import org.hawkular.apm.api.model.analytics.ServiceDeployment;
+
+/**
+ * Provides access to Service Deployment information. The name of this class is a bit unfortunate, but follows the pattern
+ * for this kind of class.
+ *
+ * @author Juraci Paixão Kröhling
+ */
+public interface ServiceDeploymentService {
+
+    /**
+     * Returns the available service deployments. Service here, at this point, are individual services. A service has 1+ buildStamps.
+     * The list of buildStamps within the service is <b>not</b> comprehensive, but limited to the 5 most recently created stamps.
```

Comment:
As mentioned, not sure we should limit the number as this could be controlled based on a timespan on the query criteria.